source/dotnet/Library/AdaptiveCards/Rendering/TextBlockConfig.cs

```diff
@@ -34,6 +34,12 @@ public class TextBlockConfig
         ///  Is it allowed for the text to wrap
         /// </summary>
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public bool Wrap { get; set; }       
+        public bool Wrap { get; set; }
+
+        /// <summary>
+        /// The maximum width for text
+        /// </summary>
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public uint MaxWidth { get; set; }
```

Comment:
While `uint` is a better type I'm pretty sure we've been using `int` everywhere for this. Up to you